### PR TITLE
docs: useful notes for testnetify

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

Documenting context I found useful when using the new in-place testnet creator https://github.com/osmosis-labs/osmosis/pull/7374

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -675,3 +675,15 @@ What it does:
...
+
+### Useful Notes
+For our in-place testnet command (`osmosisd in-place-testnet ...`), if you rely on setting up a testnet from mainnet state,
+where future module stores have not been added yet, hardcode the following in `NewOsmosisApp` in `app.go`:
+
+```go
+// One height above the latest mainnet snapshot height.
+app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(13553474, &v23.Upgrade.StoreUpgrades))
+```
+This allows creating a testnet from mainnet state where the new module store is properly registered.
```
Comment (mattverse) [inline CONTRIBUTING.md:689]:
```suggestion
This allows creating a testnet from mainnet state where the new module store is properly registered.

Note that this is only necessary only when there is a new module store being registered in the upcoming major upgrade
```

**Review by mattverse — APPROVED** [review]
Nice! Thanks 🌮

**Comment by czarcas7ic** [comment]
I am going to try and automate this today to prevent this from being necessary. Thanks for finding this / describing the work around

**Comment by czarcas7ic** [comment]
@p0mvn really appreciate this feedback. I think we can close now that I was able to automate this!

**Pull request closed without merging**